### PR TITLE
Add test for loadbalancerconfig

### PR DESCRIPTION
**Pull request by niuzhenguo**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: pkg/config/loadbalancerconfig_test.go [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+
+[NetworkingOptions]
+public-network-name=` + publicNetworkName + `1
+public-network-name=` + publicNetworkName + `2
+internal-network-name=` + internalNetworkName + `1
+internal-network-name=` + internalNetworkName + `2
+
+[MetadataOptions]
+search-order=` + searchOrder + `
+request-timeout=` + metadataRequestTimeout + `
```
Comment (chengxiangdong) [inline pkg/config/loadbalancerconfig_test.go:106]: this option has been removed and this line is no longer needed

Comment (niuzhenguo) [reply pkg/config/loadbalancerconfig_test.go]: done

**Pull request merged**